### ISSUE TITLE
compile failed error Qt requires a C++11 compiler and yours does not seem to be that.

### ISSUE THREAD

**Issue by wenkuiwang** [mltframework/shotcut#359]
I clone the project to local.and try compile the project  follow  the step:
   1)  qmake PREFIX=/opt/Qt5.7.0/
   2) make
but it failed.
```
cd CuteLogger/ && ( test -e Makefile || /usr/lib/x86_64-linux-gnu/qt5/bin/qmake  *******/shotcut/CuteLogger/CuteLogger.pro PREFIX=/opt/Qt5.7.0/ -o Makefile ) && make -f Makefile 
make[1]: Entering directory  *******/shotcut/CuteLogger'
g++ -c -m64 -pipe -O2 -fPIC -Wall -W -D_REENTRANT -DCUTELOGGER_LIBRARY -DQT_NO_DEBUG -DQT_CORE_LIB -I. -Iinclude -isystem /usr/include/x86_64-linux-gnu/qt5 -isystem /usr/include/x86_64-linux-gnu/qt5/QtCore -I. -I/usr/lib/x86_64-linux-gnu/qt5/mkspecs/linux-g++-64 -o Logger.o src/Logger.cpp
In file included from /usr/include/x86_64-linux-gnu/qt5/QtCore/qglobal.h:83:0,
                 from /usr/include/x86_64-linux-gnu/qt5/QtCore/qchar.h:43,
                 from /usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:48,
                 from /usr/include/x86_64-linux-gnu/qt5/QtCore/QString:1,
                 from include/Logger.h:22,
                 from src/Logger.cpp:15:
/usr/include/x86_64-linux-gnu/qt5/QtCore/qcompilerdetection.h:562:6: error: #error Qt requires a C++11 compiler and yours does not seem to be that.
 #    error Qt requires a C++11 compiler and yours does not seem to be that
```

so I append   "CONFIG += c++11"  to the pro files for qmake:
     ./src/src.pro
    ./mvcp/mvcp.pro
    ./shotcut.pro
    ./CuteLogger/CuteLogger.pro
then compile success,
but  execute the command  ./src/shotcut
it Segmentation fault 

can anyone help me set up the project ?

**Comment by ddennedy** [comment]
You can use or reference the build script at scripts/build-shotcut.sh. This is only used for tracking defects, and this is not one, but I will continue to allow comments although I cannot promise I will be available to help you. Perhaps if you explain why you are trying to build it yourself and your intention, I may be more motivated to help, but also realize that I have very limited attention to give as well.

**Comment by wenkuiwang** [comment]
i  am trying to build the project for developr custom functions. I will try use scripts/build-shotcut.sh ,
but it's not available for  Qt version 5.7.1. 
 I  have found  run crash reason, the executable file can't read qml files. but when I install the project,it run success. 
  @ddennedy do you hava any  Tutorial for run the project in qt creator .
